src/transformers/models/whisper/processing_whisper.py

```diff
@@ -91,3 +96,22 @@ def decode(self, *args, **kwargs):
         the docstring of this method for more information.
         """
         return self.tokenizer.decode(*args, **kwargs)
+
+    def get_prompt_ids(self, text: str):
+        """Converts prompt text to IDs that can be passed to [`~WhisperForConditionalGeneration.generate`]."""
+        _verify_no_special_tokens(text, self.tokenizer.all_special_tokens)
+        prompt_token = "<|startofprev|>"
+        batch_encoding = self.tokenizer(prompt_token, text.strip(), add_prefix_space=True, add_special_tokens=False)
+        batch_encoding.convert_to_tensors(tensor_type="pt")
```

Comment:
To make this method compatible with TF and JAX in the future, could we add an argument `return_tensors` to this method and set the tensor type based on that? For consistency with the library, the argument `return_tensors` should default to `np` (NumPy).